src/pages/Location/Location.js

```diff
@@ -156,30 +162,33 @@ const LocationComponent = () => {
         snackOpen: true,
         snackMessage: 'Your location has been saved.',
       }));
-      callCoverCropApi(`https://phzmapi.org/${zipCode}.json`)
-        .then((response) => {
-          let { zone } = response;
-
-          if (zone !== '8a' && zone !== '8b') {
-            zone = zone.slice(0, -1);
-          }
-
-          if (selectedFieldIdRedux && selectedUserField.id === selectedFieldIdRedux && regionShorthandRedux) {
-            // if there exists available region from user history api, set it as user history value
-            setRegionShorthand(regionShorthandRedux);
-          } else if (councilShorthandRedux !== 'MCCC') {
-            setRegionShorthand(zone);
-          } else {
-            // if council is MCCC, change selectedRegion based on county
-            setRegionShorthand(county.replace(' County', ''));
-          }
-        })
-        .catch((err) => {
-          // eslint-disable-next-line no-console
-          console.log(err);
-          // for places where api didn't work, set region to default.
-          setRegionShorthand(regionsRedux[0].shorthand);
-        });
+      if (councilShorthandRedux === 'MCCC') {
+        // if council is MCCC, change selectedRegion based on county
+        setRegionShorthand(county.replace(' County', ''));
```

Comment:
this logic was not changed just moved. the error was present before these updates. i think it has to do with not having handled Canada yet. I will take a look and see if i can make a quick fix to prevent the error.